config/rbac/role.yaml

```diff
@@ -12,23 +12,17 @@ rules:
   - services
   verbs:
   - create
-  - delete
   - get
   - list
-  - patch
-  - update
   - watch
 - apiGroups:
   - apps
   resources:
   - deployments
   verbs:
   - create
-  - delete
   - get
   - list
-  - patch
```

Comment:
what if the image changes and we need to update the deployment

Reply:
then we update the role and update the deployment

currently if the deployment exist I leave it alone. are there more tests we would like to do on the deployment?